src/current/cockroachcloud/use-managed-service-backups.md

```diff
@@ -139,10 +139,18 @@ To restore a cluster:
 
     The **Restore cluster** module displays with backup details.
 
+1. Select the cluster to restore to. You can restore to: a) the same cluster or b) a different cluster. By default, the option shows the current cluster. The dropdown displays options to restore to a different cluster. 
+
+    {{site.data.alerts.callout_info}}
+    Only active clusters are displayed. You can perform a cross-cluster restore across clusters that belong to the same organization. Incompatible versions cannot be selected and restoring {{ site.data.products.dedicated }} to {{ site.data.products.serverless }} or vice versa does not work. 
+    {{site.data.alerts.end}}
+
 1. Click **Continue**.
 
+1. Enter the name of the destination cluster. 
+
     {{site.data.alerts.callout_danger}}
```

Comment:
From a security perspective, I'd like to see this warning higher up (maybe at the top of the "restore" section). I'd also like to see it noted that *because* of how destructive and disruptive this operation is (anyone who has it or can get the API for a service account that has can hurt your business reeeal bad), it should be performed extremely carefully, and the roles that are capable of performing it should be restricted with extreme care according to the POLP https://en.wikipedia.org/wiki/Principle_of_least_privilege

Reply:
Also, for orgs that have opted in to fine-grained access--and very soon for all orgs--it is now possible to perform these operations with the new Cluster Operator and Cluster Admin role, as well as Org Admin and the deprecated Org admin (legacy)--which corresponds to what is now Org Admin (yeah, this is confusing...).
Could you please add that? I.e., on line 118, instead of "Org Administrators..." it should say like "Users with the Org Administrator, Org Administrator (legacy), Cluster Operator or Cluster Admin roles can..."
Here is where the links to those roles are:
https://www.cockroachlabs.com/docs/cockroachcloud/authorization#cluster-operator

For context, the explanation about the access model transition is at the top of the page there: https://www.cockroachlabs.com/docs/cockroachcloud/authorization